rust/sbp2json/src/lib.rs

```diff
@@ -78,7 +79,13 @@ where
     W: Write,
     F: Formatter + Clone,
 {
-    let source = maybe_fatal_errors(sbp::iter_messages(input), fatal_errors);
+    let source = maybe_fatal_errors(sbp::iter_frames(input), fatal_errors).map(|frame| {
+        frame.to_sbp().unwrap_or(Sbp::Unknown(Unknown {
+            msg_id: 0,
+            sender_id: None,
```

Comment:
you can set these two fields with `frame.msg_type()` and `frame.sender_id()`

Reply:
hmmm how would we distinguish if the msg is an unknown if we pass the msg type as well
would just be a secret corrupt msg when parsing but just looks like its not corrupt 🤔